src/main/resources/fxml/teams.fxml

```diff
@@ -1,21 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import java.lang.*?>
-<?import java.util.*?>
-<?import javafx.scene.*?>
-<?import javafx.scene.control.*?>
-<?import javafx.scene.layout.*?>
+<?import javafx.scene.layout.AnchorPane?>
 
```

Comment:
looks like it is overwriting somthing that somone did... care to explain? everything else looks perfect

Reply:
I didn't need all this because I created the objects (table view and lable) in code, not in the FXML.